src/components/Pf/components/form/PfForm.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+import { PFVariables } from '../../PFVariables';
+import { hyphenate } from '../../../../utils/ComponentUtils';
+
+interface PfVAlignFormProps {
+    title: string;
+    indx: string;
+    showAsRequired?: boolean;
+    showInputField?: boolean;
+}
+
+export const VAlignForm: React.FunctionComponent<PfVAlignFormProps> = (props) => {
+    return (
+        <form noValidate className={ PFVariables.Form }>
+            <div className={ PFVariables.FormGroup }>
+                <label className={ PFVariables.FormLabel } htmlFor={ hyphenate(PFVariables.VerticalAlignLabels, props.indx) }>
+                    <span className={ PFVariables.FormLabelText }>{props.title}</span>
+                    { props.showAsRequired &&
+                       <span className={ PFVariables.FormLabelRequired } aria-hidden="true">&#42;</span> }
+                </label>
+                { props.showInputField && <input className={ PFVariables.FormControl } type="text"
+                    id={ hyphenate(PFVariables.VerticalAlignLabels, props.indx) } name={ hyphenate(PFVariables.VerticalAlignLabels,  props.indx) }
+                    required/>}
+            </div>
+        </form>
+    );
```

Comment:
One of my concerns here is that we have to use a Form component to display a title here, in this case is OK.

Also, could we have used the [Form](https://patternfly-react.surge.sh/documentation/react/components/form), `FormGroup` (see examples in Form) and `label` Components directly to build this special `Title` component? 

```
<Form className="extra-class-names-that-might-be-missing">
  <FormGroup className="extra-class-names-that-might-be-missing">
      <label className={ PFVariables.FormLabel }>
         <span className={ PFVariables.FormLabelText }>{props.title}</span>
      </label>
  </FormGroup>
</Form>
```

If not, I don't think we need to make this too general, unless there is a special className the `Form` component is missing (and we want to use it on all the places), if so, we can create a PFForm (only an example) and have it use the children prop to make it more reusable.

e.g.
```tsx
const PFFormWithSecretClass = () => {
    return <Form className="secret-class-name"> { props.children } </Form>;
}
```